mne/channels/montage.py

```diff
@@ -348,6 +347,54 @@ def get_positions(self):
         )
         return positions
 
+    @verbose
+    def add_estimated_fiducials(self, subject, subjects_dir, verbose=None):
+        """Estimate fiducials based on FreeSurfer ``fsaverage`` subject.
+
+        This takes a montage with the ``mri`` coordinate frame,
+        corresponding to the FreeSurfer RAS (xyz in the volume) T1w
+        image of the specific subject. It will call
+        :func:`mne.coreg.get_mni_fiducials` to estimate LPA, RPA and
+        Nasion fiducial points.
+
+        Parameters
+        ----------
+        %(subject)s
+        %(subjects_dir)s
+        %(verbose)s
+
+        See Also
+        --------
+        :ref:`tut-plot_source_alignment`
```

Comment:
This is how it is in `plot_source_alignment.py`
```suggestion
        :ref:`plot_source_alignment`
```

Reply:
(will push a commit for this plus a couple of tiny changes)